zk_toolbox/crates/zk_inception/src/commands/chain/init.rs

```diff
@@ -160,3 +165,36 @@ async fn register_chain(
     contracts.set_chain_contracts(&register_chain_output);
     Ok(())
 }
+
+fn apply_port_offset(port_offset: u16, general_config: &mut GeneralConfig) -> anyhow::Result<()> {
```

Comment:
we have `update_ports` function in `crates/config/general.rs` counf we unify these functions?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/5ca9cf1d62d831447e4b712cf3d3ced1b934b1e5